src/content/docs/infrastructure/amazon-integrations/manage-aws-data/aws-integrations-metrics.mdx

```diff
@@ -4574,6 +4595,162 @@ We no longer add these mappings, and we're committed to preserving all the new m
+</Callout>
+
+    </Collapser>
+</CollapserGroup>
+
+## Create alert conditions [#set-alerts]
+
+You can create NRQL alert conditions on metrics from a metric stream. Make sure your filter limits data to metrics from CloudWatch Metric Stream only. To do that, construct your queries like this:
+
+```sql
+SELECT sum(aws.s3.5xxErrors) FROM Metric WHERE collector.name = 'cloudwatch-metric-streams' FACET aws.accountId, aws.s3.BucketName
+```
+
+To make sure your alerts process the data correctly, configure advanced signal settings. Advance signal settings addresses how AWS CloudWatch receives metrics from services with a certain delay. For example, Amazon guarantees that 90% of EC2 metrics are available in CloudWatch within 7 minutes of them being generated. When you add streaming metrics from AWS to New Relic, however, it can add up to 1 minute additional delay as data needs to buffer in Amazon Firehose.
+
+To configure the signal settings, go to <DNT>**Condition Settings**</DNT> and click on <DNT>**Advanced Signal Settings**</DNT>. Enter the following values:
+
+    1. <DNT>**Aggregation window**</DNT>. We recommend setting it to 1 minute. If you're having issues with flapping alerts or alerts not triggering, consider increasing it to 2 minutes.
+    1. <DNT>**Offset evaluation by**</DNT>. Depending on the service, CloudWatch may send metrics with a certain delay. The value is set in windows. With a 1-minute aggregation window, setting the offset to 8 ensures the majority of the metrics are evaluated correctly. You may be able to use a lower offset if the delay introduced by AWS and Firehose is less.
+    1. <DNT>**Fill data gaps with**</DNT>. Leave this void, or use Last known value if gaps in the data coming from AWS lead to false positives or negatives.
+
+See our documentation on [how to create NRQL alerts](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/) for more details.
+
+## Tags collection [#tags-collection]
+
+New Relic provides enhanced dimensions from metrics coming from AWS CloudWatch metric streams. Resource and custom tags are automatically pulled from most services and are used to decorate metrics with additional dimensions. Use [metrics and events](/docs/query-your-data/explore-query-data/explore-data/introduction-data-explorer/) to see which tags are available on each AWS metric.
+
+The following query shows an example of tags being collected and queried as dimensions in metrics:
+
+```
+SELECT average(`aws.rds.CPUUtilization`) FROM Metric FACET `tags.mycustomtag` SINCE 30 MINUTES AGO TIMESERIES
+```
+
+Not all metrics have their custom tags as dimensions. Only metrics linked to entities listed in the [entity explorer](/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-one-entity-explorer) have their custom tags associated. The AWS CloudWatch metric stream doesn't include tags as part of the stream message; hence, additional processing is required on the New Relic side.
```

Comment:
Isn't "Metric Stream" in the third sentence capitalized elsewhere? I'm not sure which is correct but it should be consistent I think!